client.go

```diff
@@ -368,7 +379,13 @@ func (c *Client) readRaw(ctx context.Context, recordID string) (*Record, error)
 
 // Read reads a record given a record ID, decrypts it, and returns the result.
 func (c *Client) Read(ctx context.Context, recordID string) (*Record, error) {
-	record, err := c.readRaw(ctx, recordID)
+	return c.ReadFields(ctx, recordID, nil)
+}
+
+// ReadFields reads a record given a record ID, selecting a subset of fields,
+// and returns a decrypted result.
+func (c *Client) ReadFields(ctx context.Context, recordID string, fields []string) (*Record, error) {
```

Comment:
Oh golang, no default args _or_ overloading... `ReadFields` is the next best thing, I guess. Good call.